### PR TITLE
fix: set ledger_index to `validated`

### PR DESCRIPTION
**Pull request by natenichols**
## High Level Overview of Change
There were a couple of places where we were assuming the default value for `ledger_index` was validated. This PR explicitly specifies `ledger_index: validated` in those places.

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

**Review by mvadari — COMMENTED** [review]
Can you do it in `autofill` too?

**Review by mvadari — APPROVED** [review]

**Comment by mvadari** [comment]
quick question: your commit message says "remove unneeded ledgerAccept", but your code diff shows that you added a `ledgerAccept`

File: test/integration/transactions/paymentChannelClaim.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { generateFundedWallet, testTransaction } from '../utils'
 // how long before each test case times out
 const TIMEOUT = 20000
 
-describe('PaymentChannelFund', function () {
+describe('PaymentChannelClaim', function () {
```
Comment (JST5000) [inline test/integration/transactions/paymentChannelClaim.ts:16]: Oops, good catch!

**Review by JST5000 — APPROVED** [review]

**Comment by natenichols** [comment]
@mvadari  I messed up that commit message.

**Pull request merged**